src/main/scala/com/wavesplatform/Coordinator.scala

```diff
@@ -134,6 +142,13 @@ class Coordinator(
       Left(GenericError("Checkpoint already applied"))
     }
 
+  def processRollback(blockId: ByteStr): Either[ValidationError, BigInt] = {
```

Comment:
Wouldn't `.right.map(_=>history.score())` do the job? if score is needed, it can be moved deeper into `blockchainUpdater.removeAfter`